src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+
+function sortList() {
+  const arrPeople = [...list].map(el => {
```

Comment:
map doesn't mutate array

```suggestion
  const arrPeople = list.map(el => {
```